slack_sdk/models/blocks/block_elements.py

```diff
@@ -172,17 +172,12 @@ def _validate_action_id_length(self) -> bool:
         )
 
 
+# This is a base class
 class InputInteractiveElement(InteractiveElement, metaclass=ABCMeta):
     placeholder_max_length = 150
 
     attributes = {"type", "action_id", "placeholder", "confirm"}
 
-    def _subtype_warning(self):
```

Comment:
This is already defined in the super class. We can safely remove this overridden one